### ISSUE TITLE
PDController: gdPowerPlant and tmSOSystem

### ISSUE THREAD

**Issue by balacij** [JacquesCarette/Drasil#2446]
Related to #2371

Currently, these 2 chunks are not given any sort of "left-hand side" of their respective equations so I'm not quite sure what kind of a ModelKind model I should be creating for either of them.

![image](https://user-images.githubusercontent.com/1627302/118054124-b37e7a00-b353-11eb-936e-399776b6bca6.png)

![image](https://user-images.githubusercontent.com/1627302/118054103-a95c7b80-b353-11eb-8b85-689fefb781b7.png)

**Comment by balacij** [comment]
I hope it's okay that I assigned both of you @JacquesCarette and @smiths.

**Comment by JacquesCarette** [comment]
Yes, that's fine. In this case, I think it's going to be @smiths who needs to have first crack at it. Certainly these are not 'equations'. But what is the thing being defined here, I don't know.

**Comment by smiths** [comment]
I can look at the example, but it might be a bit until I get to it.  The PDController example hasn't really had a very detailed review to this point.  (This example didn't come to us by the normal channels.)  The equations will need to introduce a symbol for the transfer function, but I'll have to look to see if that symbol is used elsewhere or if we'll have to introduce it in multiple places.

**Comment by balacij** [comment]
Thank you! That would be perfect. No rush, I have many tasks to work on in the mean time.

**Comment by balacij** [comment]
I can give this a go too if you don't mind validating my work @smiths.
